modules/tensor_mechanics/src/materials/ComputeRSphericalFiniteStrain.C

```diff
@@ -87,3 +87,13 @@ ComputeRSphericalFiniteStrain::computeProperties()
     computeQpStrain();
   }
 }
+
+void
+ComputeRSphericalFiniteStrain::computeQpIncrements(RankTwoTensor & total_strain_increment,
+                                                   RankTwoTensor & rotation_increment)
+{
+  for (unsigned int i = 0; i < 3; ++i)
+    total_strain_increment(i, i) = std::log(_Fhat[_qp](i, i));
+
+  rotation_increment.setToIdentity();
+}
```

Comment:
shall we add this function to `ADComputeRSphericalFiniteStrain` too?

Reply:
I've added that to my latest changes.